public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -370,6 +380,36 @@ function getVariables(oldDash: DashboardSceneState) {
   return variables;
 }
 
+function getAnnotations(state: DashboardSceneState): AnnotationQueryKind[] {
+  const data = state.$data;
+  if (!(data instanceof DashboardDataLayerSet)) {
+    return [];
+  }
+  const annotations: AnnotationQueryKind[] = [];
+  for (const layer of data.state.annotationLayers) {
+    if (!(layer instanceof dataLayers.AnnotationsDataLayer)) {
+      continue;
+    }
+    const result: AnnotationQueryKind = {
+      kind: 'AnnotationQuery',
+      spec: {
+        name: layer.state.query.name,
+        datasource: layer.state.query.datasource!, // FIXME: this cannot be undefined
+        query: {
+          kind: layer.state.query.datasource?.type ?? 'default',
+          spec: layer.state.query,
+        },
+        enable: Boolean(layer.state.isEnabled),
+        hide: Boolean(layer.state.isHidden),
+        filter: layer.state.query.filter ?? defaultAnnotationPanelFilter(),
+        iconColor: layer.state.query.iconColor,
```

Comment:
We need an indiction here as well to say whether or not a annotation is built in. Currently we handle this on the query level (there's a property called `builtIn` on the default annotations provided by grafana). I think we need to lift it to AnnotationQueryKind spec.